daemon/cmd/daemon_main.go

```diff
@@ -1254,6 +1257,10 @@ func initEnv(vp *viper.Viper) {
 		log.Warn("IPSec encrypted overlay is enabled but IPSec is not. Ignoring option.")
 	}
 
+	if option.Config.EnableRemoteNodeMasquerade && !option.Config.EnableBPFMasquerade {
+		log.Fatal("EnableRemoteNodeMasquerade is enabled but EnableBPFMasquerade is not. Ignoring option.")
+	}
+
```

Comment:
```suggestion
	if option.Config.EnableRemoteNodeMasquerade && !option.Config.EnableBPFMasquerade {
		log.Warn("EnableRemoteNodeMasquerade is enabled but EnableBPFMasquerade is not. Ignoring option.")
	}

```


Right now we `Fatal()`. If we want to ignore the misconfig & continue, then you'd want a `Warn()` instead.

Reply:
We have discussed this in another thread and it was originally warn() @tamilmani1989 asked to make it a fatal()